### PR TITLE
Revamp README

### LINKED ISSUE #795: Revamp README files
1. Move the stock Girder4 README content somewhere else (`DEVELOPMENT.md`?)
2. Describe in brief the content in this repository (i.e., Django backend + Vue frontend)
3. Other marketing-type material
4. Links to other READMEs (i.e., Vue app)

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #795.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,122 +1,28 @@
...
-2. Run:
-   1. `source ./dev/export-env.sh`
-   2. `./manage.py runserver`
-3. Run in a separate terminal:
-   1. `source ./dev/export-env.sh`
-   2. `celery --app dandiapi.celery worker --loglevel INFO --without-heartbeat -Q celery,calculate_sha256 -B`
-4. When finished, run `docker-compose stop`
+To get help:
+- ask a question: https://github.com/dandi/helpdesk/discussions
+- file a feature request or bug report: https://github.com/dandi/helpdesk/issues
```
Comment (satra) [inline README.md:24]:
```suggestion
- file a feature request or bug report: https://github.com/dandi/helpdesk/issues/new/choose
```

**Comment by satra** [comment]
overall lgtm. 

would it be useful to add frontend developement to the developer docs as well. it wasn't mentioned and not part of the current docker compose.

**Review by mvandenburgh — COMMENTED** [review]
The CI badge in https://github.com/dandi/dandi-archive/blob/revamp-readme/web/README.md is still pointing at the old repo - it will either need to be updated to the new repo (or maybe just remove it, is anyone getting value out of having it there?)

**Comment by waxlamp** [comment]
> The CI badge in https://github.com/dandi/dandi-archive/blob/revamp-readme/web/README.md is still pointing at the old repo - it will either need to be updated to the new repo (or maybe just remove it, is anyone getting value out of having it there?)

Good catch. I updated it rather than remove it (we still can, but this PR is all about minimally fixing up the READMEs). See 502234b.

**Comment by waxlamp** [comment]
> overall lgtm.
> 
> would it be useful to add frontend developement to the developer docs as well. it wasn't mentioned and not part of the current docker compose.

Yes, definitely would be useful. However, I'm working with our comm people now to begin a comprehensive overhaul of our docs, so how to present the basics of development guidelines via the READMEs will be part of that effort.

File: README.md [hunk 2]
```diff
@@ -1,122 +1,28 @@
...
-Then go to `/swagger` and use `GET /auth/token` end-point.
-
-### Supplying the Token
-In API endpoint calls, add the `Authorization` HTTP header with a value of
-`Token <token_value>`. For example, for a token `1234`, include the header:
-`Authorization: Token 1234`.
-
-## Custom management scripts
-Documentation for the various custom `manage.py` scripts can be found in `DEVELOPMENT.md`.
+To hack on the archive codebase:
```
Comment (satra) [inline README.md:27]:
```suggestion
To hack on the archive django codebase:
```

Comment (waxlamp) [reply README.md:27]: I went a step further with the change in f923cf4; let me know what you think.

**Review by mvandenburgh — APPROVED** [review]

**Review by satra — APPROVED** [review]

**Comment by satra** [comment]
@waxlamp - i approved it, but i haven't actually worked through the steps to verify.

**Comment by waxlamp** [comment]
> @waxlamp - i approved it, but i haven't actually worked through the steps to verify.

This was largely about restructuring the main README to be more encompassing of the two software pieces in the repo. We'll be making lots of changes/improvements once Mary Elise from Kitware's comm team has a change to get engaged.

**Pull request merged**